### ISSUE TITLE
Add support for ExtensionProperty parseValue option (Open API v2)

### ISSUE THREAD

**Issue by donaltuohy** [swagger-api/swagger-core#3375]
Hi.

I am trying to add a custom key:value option to the annotations of my models. I have used the Extension properties option to add custom key:values it but the types are restricted to String. I want to add `"x-nullable": true` but because the type is restricted to String, it renders the value option as a string: `"x-nullable": "true"`. which causes a lot of problems for us.

What is currently rendered: 
```
    "Release" : {
      "type" : "object",
      "properties" : {
        "delay" : {
          "x-nullable" : "true"  // Note - boolean is actually a string
        }
      },
    },
```

What we need:
```
    "Release" : {
      "type" : "object",
      "properties" : {
        "delay" : {
          "x-nullable" : true // Is actual Boolean
        }
      },
    },
```

I see in issue https://github.com/swagger-api/swagger-core/issues/2723 , support for a parseValue option (on custom extension properties) was added for the openApi 3 library. We are using the openApi2 library so would it possible to have this change added to Swagger Core 1.5.X version?

Thank you 🙏

**Comment by donaltuohy** [comment]
I tried to submit a PR myself but I do not have the rights :/ 

```
ERROR: Permission to swagger-api/swagger-core.git denied to donaltuohy.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists
```

**Comment by webron** [comment]
@donaltuohy it looks like you tried pushing directly to repository, which would not work. You need to fork the repo, make the changes, and then submit a PR. See https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork.

**Comment by donaltuohy** [comment]
Hey @webron thanks a mill for sharing that. I created the PR here: https://github.com/swagger-api/swagger-core/pull/3377

**Comment by frantuma** [comment]
Thanks for reporting and for the PR! implemented in #3377
